frontend/src/components/QrCode/FigureQrCode.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="figure-qr-code">
+    <div class="qrbox">
+      <q-r-canvas :options="options" class="canvas" />
+    </div>
+  </div>
+</template>
+<script>
+import { QRCanvas } from 'qrcanvas-vue'
+
+export default {
+  name: 'FigureQrCode',
+  components: {
+    QRCanvas,
+  },
+  props: {
+    text: { type: String, required: true },
+  },
+  data() {
+    return {
+      options: {
+        cellSize: 8,
+        correctLevel: 'H',
+        data: this.text,
+      },
+    }
+  },
+  created() {
+    const image = new Image()
+    image.src = 'img/gdd-coin-sw.png'
```

Comment:
this image is still to big

Reply:
du meinst das logo, der coin? oder der qr code?

Reply:
The image is appropriate and not to big in my eyes.